x/staging/subspaces/types/msgs.go

```diff
@@ -32,11 +32,11 @@ func (msg MsgCreateSubspace) ValidateBasic() error {
 	}
 
 	if !IsValidSubspace(msg.SubspaceID) {
-		return sdkerrors.Wrap(ErrInvalidSubspaceID, "subspace id must be a valid SHA-256 hash")
+		return ErrInvalidSubspaceID
 	}
 
 	if strings.TrimSpace(msg.Name) == "" {
-		return sdkerrors.Wrap(ErrInvalidSubspaceName, "subspace name cannot be empty or blank")
+		return ErrEmptySubspaceName
```

Comment:
I think we can use a more generic `ErrInvalidSubspace` instead here, to make it coherent with the other errors
```suggestion
		return sdkerrors.(ErrInvalidSubspace, "name cannot be empty or blank")
```